meta-ov/recipes-apps/ovmenu-ng-skripts/files/download-all.sh

```diff
@@ -7,13 +7,12 @@ if [ ! -d "$USB_PATH" ]; then
 	mkdir -p "$USB_PATH"
 fi
 
-if [ -z "$(ls $DOWNLOAD_PATH/* 2>/dev/null)" ]; then
+if [ -z "$(find "$DOWNLOAD_PATH" -type f 2>/dev/null)" ]; then
```

Comment:
Insert a `|head -1` so `find` gets killed with `SIGPIPE` after finding the first file; with huge directory trees, that will be a big speedup.

Reply:
good point. Will do.

Reply:
done